build-system/tasks/compile.js

```diff
@@ -252,7 +251,7 @@ function compile(entryModuleFilenames, outputDir,
       'third_party/webcomponentsjs/ShadowCSS.js',
       'third_party/rrule/rrule.js',
       'third_party/react-dates/bundle.js',
-      'node_modules/dompurify/dist/purify.cjs.js',
+      'node_modules/dompurify/dist/purify.es.js',
```

Comment:
Is this change part of moving to the new version or just convenience since there are already changes here.

Reply:
I was having some trouble getting both Babel and CC to work when importing the CJS version. The ES version didn't have this issue but I'm not sure why.

Reply:
Makes sense, thanks.